.github/workflows/sync-release-to-gitcode.yml

```
@@ -0,0 +1,447 @@
+  GITCODE_OWNER: ${{ vars.GITCODE_OWNER || 'mkdir700' }}
+  GITCODE_REPO: EchoPlayer
+
+jobs:
+  sync-to-gitcode:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Fetch all history for all branches and tags
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)